app/models/ansible_role.rb

```diff
@@ -38,4 +39,14 @@ class AnsibleRole < ApplicationRecord
   class Jail < Safemode::Jail
     allow :name
   end
+
+  def clone_to_org!(new_org_id = nil)
+    cloned = deep_clone
+    cloned.organization_id = new_org_id if new_org_id
+    cloned.save!
+    ansible_variables.each do |variable|
+      cloned.ansible_variables << variable.deep_clone(except: :ansible_role_id, include: :lookup_values)
```

Comment:
Is cloning overrides a good idea? We might end up with cloned roles in OrgB that have overrides for host in OrgA...

Reply:
We might, but I'm not sure how else to approach it, droping the overrides sounds worse then having useless ones.

Reply:
I am more afraid of the cloned ones not being useless and that their value will actually be applied since they are not taxable. I'll need to thoroughly test that.